packages/grida-canvas-sdk-render-figma/lib.ts

```
@@ -0,0 +1,722 @@
+
+  const context = buildContext({
+    node_id_generator: () => `refig-${++counter}`,
+  });
+
+  const individualResults = rootNodes.map((rootNode) =>
+    iofigma.restful.factory.document(rootNode as any, {}, context)
+  );
+
+  const imageRefsUsed = new Set<string>();
+  for (const r of individualResults) {
+    r.imageRefsUsed.forEach((ref: string) => imageRefsUsed.add(ref));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)